### ISSUE TITLE
Configurable menu bar

### ISSUE THREAD

**Issue by Grinnz** [Grinnz/perldoc-browser#21]
The elements and ordering on the menu bar should be configurable. This can easily be done with two array config options for left and right placement.

**Comment by kiwiroy** [comment]
Would this include support for perlbrew `local::lib` libs?

**Comment by Grinnz** [comment]
It would not. These are intentionally not supported because they can only work for one Perl, and the user running perldoc-browser might have one enabled that does not apply. Supporting it would require configuration for extra libs for each Perl, but feel free to open an issue if it's a feature you want.

**Comment by kiwiroy** [comment]
> Supporting it would require configuration for extra libs for each Perl, but feel free to open an issue if it's a feature you want.

Yeah, the issue is they go in `$PERLBREW_HOME/libs`, so at least that variable needs config. I might open an issue if I can find reasons other than *would be cool*.
